common/djangoapps/enrollment/views.py

```diff
@@ -634,6 +635,8 @@ def post(self, request):
                 }
             )
         except CourseEnrollmentExistsError as error:
+            log.debug('An enrollment already exists for user [%s] in course run [%s].',
```

Comment:
@ahsan-ul-haq messages logged at the DEBUG level won't appear in Splunk. You must at least log at the INFO level for logs to show up. The stack trace in this case isn't really that valuable to us in this case, so I would log at the WARNING level.